src/backend/api/meals.js

```diff
@@ -2,14 +2,84 @@ const express = require("express");
 const router = express.Router();
 const knex = require("../database");
 
-router.get("/", async (request, response) => {
+router.get("/title", async (request, response) => {
   try {
     // knex syntax for selecting things. Look up the documentation for knex for further info
-    const titles = await knex("Meal").select("title");
-    response.json(titles);
+    const rows= await knex("Meal").select("title");
+    response.json(rows);
   } catch (error) {
     throw error;
   }
 });
 
+// 1.Returns all meals
+
+router.get("/meals", async (request, response) => {
+  try {
+    // knex syntax for selecting things. Look up the documentation for knex for further info
+    const rows = await knex("Meal").select("*");
+    response.json(rows);
+  } catch (error) {
+    throw error;
+  }
+});
+
+// 2.Adds a new meal to the database
+
+router.post("/", async (req,res) =>{
+  try {
+    const rows = await knex('Meal').insert({ title: 'curd rice', description: 'curd with rice', location: 'roskilde', when: '2022-08-24', max_reservations: 4, price: 220, created_date: '2022-07-17' });
```

Comment:
Here you should acept a body  input and then do insert. This will always insert same hardcoded values

Reply:
Hej @itsgauravsaxena thank you so much for your comments, I will make the changes as per the comments thank you :)